src/Map.jsx

```diff
@@ -53,6 +54,33 @@ function sortAlphaNumeric(a, b) {
 // make a copy of routes and sort
 const sortedRoutes = muniRoutesGeoJson.features.slice(0).sort(sortAlphaNumeric);
 
+/*
+* Stop class used to handle info about selected stops
+*/
+class Stop {
+  constructor(stop) {
+    this.stop = stop;
+  }
+  isUndefined() {
+    return typeof this.stop === 'undefined';
+  }
+  setCoordinates(coordinates) {
+    this.stop = {
+      lon: coordinates[0],
+      lat: coordinates[1],
+    };
+  }
+  getCoordinateArray() {
+    return [this.stop.lon, this.stop.lat];
+  }
+  /**
+  * sees if two stops are equal by evaluating their coordinates
+  */
+  checkIfTwoStopsAreEqual(stop) {
```

Comment:
could we name this something more standard? like just `equals` so that then we can call it through `stop1.equals(stop2)`?